tests/models/test_model.py

```diff
@@ -6,6 +6,8 @@
 
 def test_fstring(selected_model):
     lm = selected_model
+    print(f"{dir(lm.engine.tokenizer)=}")
+    assert hasattr(lm.engine.tokenizer,"sp_model")
```

Comment:
We don't want to have this assert for every model -- not every tokenizer is sentencepiece based.